### PR TITLE
Update normalization

### PR DESCRIPTION
**Pull request by SteffenHeu**
- Different normalization was used in the feature table and the image chart opened from the feature table
- use all scans instead of scans where the image feature was detected in for normalization

### CODE REVIEW

File: src/main/java/io/github/mzmine/gui/preferences/MZminePreferences.java [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /*
  * Copyright (c) 2004-2022 The MZmine Development Team
- *
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/gui/preferences/MZminePreferences.java:3]: Are you using a different Copyright

**Review by robinschmid — APPROVED** [review]
Looks good to me.

**Comment by SteffenHeu** [comment]
fixed

**Comment by robinschmid** [comment]
Thanks steffen

**Pull request merged**